atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractTableMappingService.java

```diff
@@ -94,31 +94,32 @@ private TableReference getFullTableName(TableReference shortTableName) {
     private final ConcurrentHashMap<TableReference, Boolean> unmappedTables = new ConcurrentHashMap<>();
 
     @Override
-    public Set<TableReference> mapToFullTableNames(Set<TableReference> tableRefs) {
-        Set<TableReference> newSet = Sets.newHashSet();
+    public Map<TableReference, TableReference> generateMapToFullTableNames(Set<TableReference> tableRefs) {
+        Map<TableReference, TableReference> inputNameToFullTableName = Maps.newHashMapWithExpectedSize(
```

Comment:
AFAIK, the kvs is wrapped by a mapping service and the names are shortened and stored in th enamespace table, so I think `shortTableName` is more appropriate.